src/Appwrite/Utopia/Request/Filters/V13.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace Appwrite\Utopia\Request\Filters;
+
+use Appwrite\Utopia\Request\Filter;
+
+class V13 extends Filter
+{
+    // Convert 0.12 params format to 0.13 format
+    public function parse(array $content, string $model): array
+    {
+        switch ($model) {
+            // Replaced Types
+            case "database.createFloatAttribute":
+                $content = $this->convertStringToNum($content, "min");
+                $content = $this->convertStringToNum($content, "max");
+                $content = $this->convertStringToNum($content, "default");
+                break;
+            case "database.createIntegerAttribute":
+                $content = $this->convertStringToNum($content, "min");
```

Comment:
I am not sure if these were changed, please double-check

Reply:
They were changes however double-checking it appears they changed the opposite way. From string to number. I've just fixed this.